tests/test_hear_keyboard_interrupt.py

```
@@ -58,37 +66,84 @@
+			4. Verifies that the server exits with the expected status code
+
+		Success criteria:
+			- Server must exit with status code 130 (standard SIGINT exit code)
+			- Server must not exit with status code 2 (invalid arguments)
+		"""
+		theResult: bool = False
+		fail_fixture: str = str("C^INT --> HEAR == error")
+		_fixture_port_num: int = self._the_test_port
 		try:
 			self.assertIsNotNone(_fixture_port_num)
 			self.assertEqual(type(_fixture_port_num), type(int(0)))
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)